app/controllers/laa_references_controller.rb

```diff
@@ -45,4 +63,43 @@ def no_maat_id?
   def error_messages
```

Comment:
not your work i know -  but could [ditto](https://github.com/ministryofjustice/laa-court-data-ui/pull/211/files#r451485926)